third_party/istio-1.0.4/download-istio.sh

```diff
@@ -9,21 +9,31 @@ cd istio-${ISTIO_VERSION}
 # Copy CRDs template
 cp install/kubernetes/helm/istio/templates/crds.yaml ../istio-crds.yaml
 
-# Create template
+# A template with sidecar injection enabled.
 helm template --namespace=istio-system \
   --set sidecarInjectorWebhook.enabled=true \
   --set sidecarInjectorWebhook.enableNamespacesByDefault=true \
   --set global.proxy.autoInject=disabled \
   --set global.disablePolicyChecks=true \
   --set prometheus.enabled=false \
+  `# Disable mixer policy check, since in our template we set no policy.` \
+  --set global.disablePolicyChecks=true \
+  `# Set a generous number of pilot replicas to avoid Pilot being overloaded.` \
+  --set pilot.autoscaleMin=3 \
+  --set pilot.autoscaleMax=10 \
+  --set pilot.cpu.targetAverageUtilization=60 \
   install/kubernetes/helm/istio > ../istio.yaml
 
+# A liter template, with no sidecar injection.  We could probably remove
+# more from this template.
 helm template --namespace=istio-system \
   --set sidecarInjectorWebhook.enabled=false \
   --set global.proxy.autoInject=disabled \
   --set global.omitSidecarInjectorConfigMap=true \
   --set global.disablePolicyChecks=true \
   --set prometheus.enabled=false \
+  `# Disable mixer policy check, since in our template we set no policy.` \
+  --set global.disablePolicyChecks=true \
```

Comment:
Why don't we need to set `pilot.autoscaleMin1`, `pilot.autoscaleMax`, and `targetAverageUtilization` here?

Reply:
it doesn't do sidecar injection so there aren't as many proxy.  also it's supposed to be lighter weight, so I don't want to add a lot of replicas.